XamarinCommunityToolkit/Views/GravatarImageSource/GravatarHandlerUtil.shared.cs

```diff
@@ -0,0 +1,42 @@
+using System.IO;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+#if !NETSTANDARD
```

Comment:
Is this necessary?

Reply:
I could make it netstandard1.0 only... since it's not 1.0 compliant... but ultimately this is only useful on an actual platform that could implement an Image Handler

Reply:
But if the user tries to use it in a platform without support, like GTK. Will he see a good error message? I'm guessing here, but I think he will face a runtime exception telling that this method is missing... And a _correct_ message should be something like "We don't support this platform". Thoughts?

Reply:
Well this library took an explicit stance against supporting GTK so that example is a moot point. Both WPF and GTK use the same TFM until the transition to .NET 5. 

The Util class is meant simply to consolidate the logic that belongs in the platform ImageHandlers but would be common between each of them.

Reply:
well, if I'm not wrong, we can't target .NET5, because XF doesn't run on it. And when we have MAUI/.NET6 we need to see what we need to change to support both.